plugins/process/src/main/java/org/radargun/service/ProcessLifecycle.java

```diff
@@ -402,4 +399,34 @@ private String getServiceChildPIDs() {
       }
       return "";
    }
+   
+   private boolean isPIDAlive(String pid) {
+      if (pid != null) {
+         ProcessBuilder pb = new ProcessBuilder().command("ps", "-o", "pid=", "-p", pid);
```

Comment:
Well I think that you shouldn't *implement* the Windows part, as it's not implemented elsewhere either, but you shouldn't pollute the code with unix-specific stuff. I don't think that calling
```
new ProcessBuilder().command(prefix + "is-alive" + extension, pid);
```
here instead would grow the effort needed dramatically.